pawno/include/PPC_Defines.inc

```diff
@@ -240,6 +240,9 @@
 #define ADMIN_SPEC_TYPE_VEHICLE 2
 
 
+// Define strings
+#define MAX_STRING 128
```

Comment:
What do you mean by `MAX_STRING`? According to https://wiki.sa-mp.com/wiki/Limits there are two different character lenghts (Input: Chat/Commands / Output: e.g. `SendClientMessage`). I guess we should add both.

Reply:
Can you please change this to the following:

```pawn
#define MAX_STRING_INPUT_TEXT 128
#define MAX_STRING_OUTPUT_TEXT 144
```

Feel free to change it to a more suitable name if you would like to. Thanks!